### PR TITLE
frontend task #4 @theo4u

### PR DESCRIPTION
**Pull request by obamgbopa**
Corrections to the folder structure

### CODE REVIEW

**Comment by osioke** [comment]
@clux2605 you did the SVG logo yourself?

File: frontend/category.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>user-category</title>
+</head>
+<body>
+	<div id="logo">
+		<a href="index.html">
+			<img src="res/devcenter-logo.svg">
```
Comment (theoomoregbee) [inline frontend/category.html]: not found, no **res** folder instead `assets/img/`

File: frontend/explainer.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	</div>
+	<div id="content">
+		<h2>We've already done the hardwork on curating the best resources to help you.</h2>
+
+		<h3>All you have to do is try to answer the next couple of question as best as you can so we can figure out what you may like.</h3>
+
+		<h4>We then use that to create a track that might be the most engaging way for you to get into tech.</h4>
+	</div>
+	<div id="button">
+		<button type="button" href="category.html">Begin</button>
```
Comment (theoomoregbee) [inline frontend/explainer.html]: this should be a link, `<a href="category.html">Begin</a>`

File: frontend/index.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	<div id="name">
+		<h2>Coding101</h2>
+	</div>
+	<div id="content">
+		<h2>Coding101 aims to help people who are interested in coding get an head start.</h2>
+
+		<h3>We understand that coding can be a bit confusing at the beginning and we are here to help you get started.</h3>
+	</div>
+	<div id="button">
+		<button type="button" href="explainer.html">How?</button>
```
Comment (theoomoregbee) [inline frontend/index.html]: same as above, use link

File: frontend/index.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>index</title>
+</head>
+<body>
+	<div id="logo">
+		<a href="./">
+			<img src="res/devcenter-logo.svg">
```
Comment (theoomoregbee) [inline frontend/index.html]: resource not found

File: frontend/question.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+			<label>I am a Secondary School Graduate</label>
+		
+			<div class="quest-1">
+				<li>
+
+				    <h3>What do you prefer as a hobby/ what do you do in your free time</h3>
+				    <hr>
+				    
+				    <div>
+				        <input type="checkbox" name="question-1-answers" id="question-1-answers-A" value="A" />
```
Comment (theoomoregbee) [inline frontend/question.html]: the checkbox should be radio button as you can only pick one answer per question

**Comment by theoomoregbee** [comment]
check the comments placed on changed files, and no **result** page #4 last image

**Review by theoomoregbee — CHANGES_REQUESTED** [review]
1. Check your src to devcenter svg
2. No Result page
3. Use link instead of button(and buttons don't use href)

File: frontend/category.html (hunk 1 above)
Comment (obamgbopa) [reply frontend/category.html]: oh, my bad, forgot to change the dir when i changed the folder structure.

File: frontend/index.html [hunk 6]
```diff
@@ -1,25 +1,28 @@
...
-		</a>
-		
-	</div>
-	<div id="name">
-		<h2>Coding101</h2>
-	</div>
-	<div id="content">
-		<h2>Coding101 aims to help people who are interested in coding get an head start.</h2>
+    <div id="logo">
+        <a href="./">
```
Comment (theoomoregbee) [inline frontend/index.html]: no need for thr link here, anyway `href` should be **#** instead

File: frontend/index.html [hunk 7]
```diff
@@ -1,25 +1,28 @@
...
+    <div id="name">
+        <h2>Coding101</h2>
+    </div>
+    <div id="content">
+        <h2>Coding101 aims to help people who are interested in coding get an head start.</h2>
+
+        <h3>We understand that coding can be a bit confusing at the beginning and we are here to help you get started.</h3>
+    </div>
+    <div id="link">
+        <a href="explainer.html"></a>
```
Comment (theoomoregbee) [inline frontend/index.html]: no link Text

File: frontend/explainer.html [hunk 8]
```diff
@@ -1,27 +1,30 @@
...
-	</div>
-	<div id="button">
-		<button type="button" href="category.html">Begin</button>
-	</div>
+        <h3>All you have to do is try to answer the next couple of question as best as you can so we can figure out what you may like.</h3>
+
+        <h4>We then use that to create a track that might be the most engaging way for you to get into tech.</h4>
+    </div>
+    <div id="link">
+        <a href="category.html"></a>
```
Comment (theoomoregbee) [inline frontend/explainer.html]: no link text

**Review by theoomoregbee — CHANGES_REQUESTED** [review]

File: frontend/explainer.html (hunk 8 above)
Comment (obamgbopa) [reply frontend/explainer.html]: corrected

**Review by theoomoregbee — APPROVED** [review]
well done 👏

**Pull request merged**